cartoframes/carto_vl/layer/layer.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+from .query_layer import QueryLayer
+
+
+class Layer(QueryLayer):
```

Comment:
I guess this is just moving files and not doing the Layer -> Source refactor we talked about

Reply:
We're going to tackle the layers - source issue in a separate task because it needs further discussion 👍